content/blogs/newsletter-102.md

```diff
@@ -0,0 +1,113 @@
+---
+title:       "Newsletter 102 - 2022/2023 news"
+author:      "GeoB99"
+date:        2023-03-25
+tags:        [ newsletter ]
+---
+
+Hello ReactOS followers and enthusiasts! It has been quite a long time since we published the 101th newsletter and so far no further updates have been posted since then.
+While the [ReactOS Twitter account](https://twitter.com/reactos) does provide announcements, posts about working applications and such from time to time, much of what is happening with the project as a whole isn't mentioned. Most of what we are going to talk about is the current situation with releases and overall ReactOS development.
+
+## Why does it take so long to release a new version of ReactOS? Are you guys dead?
+The latest release of ReactOS is 0.4.14, published on 16th of December, 2021. That release alone took a year to be released. Back then the ReactOS followed the 3-months cadence, which every 3 months a new release would be released to the public. But since 2021, ReactOS is still at 0.4.14. Are you guys dead?
+
+The answer is certainly no, the way we handle releases has changed. Back in the day the way ReactOS releases were made were done for the sake of quantity rather than quality. Every new release after 3 months is similar to the last one, with the exceptions of a small number of features and bug fixes added to the release. But overall the net gap between the releases wasn't that big.
+For a while now we have instated a rule that for a new version to reach "Release" status, it has to have a reasonable amount of regressions (no more than 20) and that the stability isn't badly impacted by the introduction of new features or code changes done during the development.In other words, the ReactOS project is now focusing on providing quality releases, which rules out the 3-months release cadence. This approach wasn't quite feasible for a development team with a size like ours anyway.
```

Comment:
```suggestion
For a while now we have instated a rule that for a new version to reach "Release" status, it has to have a reasonable amount of regressions (no more than 20) and that the stability isn't badly impacted by the introduction of new features or code changes done during the development.
In other words, the ReactOS project is now focusing on providing quality releases, which rules out the 3-months release cadence. This approach wasn't quite feasible for a development team with a size like ours anyway.
```

Reply:
Without an extra newline, this won't show any different in the published page.

Reply:
The problem is there are two sentences not even separated with space.